src/main/java/sx/blah/discord/util/MessageHistoryRange.java

```diff
@@ -0,0 +1,160 @@
+package sx.blah.discord.util;
+
+import sx.blah.discord.handle.impl.obj.Channel;
+import sx.blah.discord.handle.obj.IChannel;
+import sx.blah.discord.handle.obj.IMessage;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MessageHistoryRange {
```

Comment:
Please document more of this class and I'll review again. I can't tell what things are meant to be doing.